src/features/dashboard/manage-dashboard/manage-dashboard.scss

```diff
@@ -3,3 +3,12 @@
     height: 90vh;
   }
 }
+
+.breadcrumbs {
+  padding-left: 5%;
+  padding-top: 1.6%;
+
+  @media screen and (max-width: 1200px) {
```

Comment:
any specific reason to mention this width ?

Reply:
yes @ahmed-deriv because at this width, there's currently the token switcher which will get removed from the header. it's at this width that it falls at the 2nd layer of the navbar

Reply:
when the header is removed, this can be adjusted and we wouldn't need this media query at all

Reply:
<img width="1190" alt="Screenshot 2024-08-08 at 1 12 39 PM" src="https://github.com/user-attachments/assets/dd3ad244-0762-4d1a-a1f5-568a45fa11a7">